fe/fe-core/src/main/java/com/starrocks/sql/analyzer/PrivilegeCheckerV2.java

```diff
@@ -1511,6 +1511,9 @@ public Void visitShowSnapshotStatement(ShowSnapshotStmt statement, ConnectContex
         public Void visitBackupStatement(BackupStmt statement, ConnectContext context) {
             checkSystemRepository(context);
             List<TableRef> tableRefs = statement.getTableRefs();
+            if (tableRefs.size() == 0) {
```

Comment:
Add the tableName in the log.

Reply:
There is no table in database. I have changed the error msg include the dbname